src/logging.h

```diff
@@ -65,6 +65,13 @@ namespace BCLog {
         BLOCKSTORE  = (1 << 26),
         ALL         = ~(uint32_t)0,
     };
+    enum class Level {
+        None = 0,
```

Comment:
What does level "None" mean? Is it the highest or lowest log level?

Reply:
> What does level "None" mean? 

I used `None` to signify that the log is an existing log that does not have any severity level.

> Is it the highest or lowest log level? This ordering will matter when we want to add, say, a priority floor per category.

You're right! Updated level of `None` to be the highest (above `Error`). This will allow us to set a priority floor in the future without accidentally removing all existing logs with `None` level.

Reply:
Thanks. I guess it's mostly a temporary worry until all log messages have an explicit level.